tests/acceptance/features/apiSpaces/quota.feature

```diff
@@ -23,7 +23,7 @@ Feature: State of the quota
   Scenario Outline: quota information is returned in the list of spaces returned via the Graph API
     Given user "Alice" has created a space "<spaceName>" of type "project" with quota "<total>"
     When user "Alice" uploads a file inside space "<spaceName>" with content "<fileContent>" to "test.txt" using the WebDAV API
-    Then for user "Alice" the JSON response should contain space called "<spaceName>" and match
+    Then for user "Alice" the JSON representation of her drive should contain space called "<spaceName>" and match
```

Comment:
i would prefer the old step rather than the new refactor since we are checking the json response from the above request.

Reply:
we are not checking the JSON response of **When**. This step makes API request for user drive information and check whether Json response contain the changes in it or not